examples/jib-multimodule/pom.xml

```diff
@@ -38,7 +38,7 @@
         <plugin>
           <groupId>com.google.cloud.tools</groupId>
           <artifactId>jib-maven-plugin</artifactId>
-          <version>1.0.0</version>
+          <version>1.0.2</version>
```

Comment:
FYI, I pushed a PR that upgrades Jib to 1.0.2 in our samples: https://github.com/GoogleContainerTools/skaffold/pull/1772